lib/msal-common/src/response/ResponseHandler.ts

```diff
@@ -425,6 +426,30 @@ export class ResponseHandler {
         );
     }
 
+    /**
+     * Calculates the number of seconds until the token expires.
+     *
+     * @param reqTimestamp - The timestamp when the request was made, in seconds.
+     * @param tokenExpirationTimestamp - The expiration timestamp of the token, which can be a number (or a number as a string), a string in ISO 8601 format, or undefined.
+     * @param returnValueIfUndefined - The value to return if the tokenExpirationTimestamp is undefined.
+     * @returns The number of seconds until the token expires, or the returnValueIfUndefined if the tokenExpirationTimestamp is undefined.
+     */
+    private getSecondsUntilTokenExpires(
+        reqTimestamp: number,
+        tokenExpirationTimestamp: number | string | undefined,
+        returnValueIfUndefined: number | undefined
```

Comment:
Do we need this as a param if we only ever pass 0?

Reply:
I added that in there because I originally coded this PR to also handle (future-proof?) ext_expires_in, refresh_in, and refresh_token_expires_in. However, I'm not sure if those are necessary, and if they are, maybe it should be another PR. I've posed this question to Bogdan and Gladwin and am awaiting feedback.

Reply:
I think even for those the fallback would be 0, no?

Reply:
refreshOnSeconds can be passed as `undefined` to `CacheHelpers.createAccessTokenEntity`, and rtExpiresOn can be passed as `undefined` to `CacheHelpers.createRefreshTokenEntity`.

Reply:
Just because it can be doesn't mean it needs to be.